src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,37 @@
+package lotto.domain;
+
+public class LottoNumber {
+
+    private final int number;
+
+    public LottoNumber(int number) {
+        this.number = validateNumber(number);
```

Comment:
유효성 검증 메서드 호출과 할당 구문을 분리하면 어떨까요?
validate 메서드에서 값을 리턴해주면 어떤 장점이 있나요?
```suggestion
        validateNumber(number);
        this.number = number;
```